common/algorithm-terms.html

```diff
@@ -35,15 +35,15 @@
     but present in the <a>input frame</a>,
     should be omitted from the output.</dd>
   <dt class="changed"><dfn>omit graph flag</dfn></dt><dd class="changed">
-    A flag that determines if framing output is always contained within a <code>@graph</code> member,
+    A flag that determines if framing output is always contained within a <code>@graph</code> <a>entry</a>,
     or only if required to represent multiple <a>node objects</a>.</dd>
   <dt><dfn>processor state</dfn></dt><dd>
     The <a>processor state</a>,
     which includes the <a>active context</a>, <a>active subject</a>, and <a>active property</a>.
     The <a>processor state</a> is managed as a stack with elements from the previous <a>processor state</a>
     copied into a new <a>processor state</a> when entering a new <a>JSON object</a>.</dd>
-  <dt><dfn>promise</dfn></dt><dd>
-    A <a data-cite="ECMASCRIPT#sec-promise-objects" class="externalDFN">promise</a> is an object that represents the eventual result of a single asynchronous operation.
+  <dt><dfn data-cite="ECMASCRIPT#sec-promise-objects">promise</dfn></dt><dd>
```

Comment:
ReSpec should pick this up and link to WebIDL automatically (not ECMAScript) - but I think it's what you want here. Note that you will need to change this everywhere in the document you talk about promises... also, avoid `<dfn>` here, just do the cross reference as per below. 
 
```suggestion
  <dt>{{Promise}}</dt><dd>
```

The `{{ WhatEver }}` syntax means "cross-reference a WebIDL interface in another spec"... ReSpec should know  know where to look.